### DISCUSSION TITLE
分享一张艾莲美图

### DISCUSSION THREAD

**Discussion by wugan-bot156** [share121/inter-knot#363] [其他]
![illust_121026135_20240730_202807](https://github.com/user-attachments/assets/534d398d-b2ed-4218-a35c-45ba188c0aca)

**Comment by Ailian-Qiao** [comment]
我是爱莲，我要告你侵权

**Comment by lbghhj** [comment]
我也是艾莲，我也要告你侵权

**Comment by xinxingjie** [comment]
我承认艾莲小姐美貌盖世无双！！！！！！

**Comment by ShuoChen08** [comment]
艾莲玉足，豪吃😋

**Comment by KamisatoAyaka-Alt** [comment]
多来点，好吃🥰
